sql/glam-fenix-dev/glam_etl/org_mozilla_fenix_glam_nightly__clients_histogram_aggregates_v1/query.sql

```diff
@@ -63,23 +59,15 @@ filtered_accumulated AS (
     histogram_aggregates
   FROM
     extracted_accumulated
-  LEFT JOIN
-    glam_etl.org_mozilla_fenix_glam_release__latest_versions_v1
-  USING
-    (channel)
   WHERE
       -- allow for builds to be slighly ahead of the current submission date, to
       -- account for a reasonable amount of clock skew
-    `moz-fx-data-shared-prod`.udf.fenix_build_to_datetime(app_build_id) < DATE_ADD(
-      @submission_date,
-      INTERVAL 3 day
-    )
+    mozfun.glam.build_hour_to_datetime(app_build_id) < DATE_ADD(@submission_date, INTERVAL 3 day)
```

Comment:
This reminded me that we can do `@submission_date + 3` now‼️

Reply:
😰